README.md

```diff
@@ -1,7 +1,42 @@
-# rds-contracts
+# RIF Marketplace Storage Pinning
 
-## message
-The `StorageProvider` can emit an arbitrary message in the function calls `setStorageOffer` and `emitMessage`. The message is is initially intended to allow the `StorageProvider` to communicate his `nodeID`, but may be used for other purposes later.
+```
+npm i @rsksmart/rif-marketplace-storage-pinning
+```
+
+## PinningManager contract
+
+### Message
+The `StorageProvider` can emit an arbitrary message in the function calls `setStorageOffer` and `emitMessage`. 
+The message is is initially intended to allow the `StorageProvider` to communicate his `nodeID`, but may be used for other purposes later.
 
 The client should be instructed to enforce the following semantics:
-- If `message` starts with `0x01`, then the following bits contain an arbitrary length `nodeID`.
\ No newline at end of file
+- If `message` starts with `0x01`, then the following bits contain an arbitrary length `nodeID`.
+
+## Glossary
+
+ - Actors
+    - **Consumer** - actor that wants his files to be persisted in the network
+    - **Provider** - actor that offers his storage for Consumers to be used in monetary exchange
+ - Entities
+    - **Offer** - Offer created by Provider that announces availability of his storage space
+      - *Billing Periods* - Specifies periods that are available to used for billing   
+      - *Billing Prices* - Specifies prices for each Billing Period, resulting in period-price tuple.
```

Comment:
Maybe add another glossary item:
*period-price tuple* - defines the Billing Price (per byte) which must be paid per Billing Period

Reply:
Well, I think that is redundant information...